### PR TITLE
Incorporating generation and running of slope in StreamCat

### PR DESCRIPTION
**Pull request by mhweber**
Changes directly in develop branch - can discuss flow of PRs in regard to develop, master, and project branches

### CODE REVIEW

File: ListPublishedUnpublishedFiles.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+local_unpublished =  [x for x  in local_list if x not in published]
+
+control = pd.read_csv(r'E:\GitProjects\StreamCat\ControlTable_StreamCat.csv')            
+local_published_metrics =  [elem.split('_Region')[0] for elem in local_published]
+local_published_metrics = list(set(local_published_metrics))
+
+list(control)
+control['Published'] = control['Final_Table_Name'].isin(local_published_metrics)
+control['Published'] = control['Published'].map({True: 'Yes', False: 'No'})
+control.to_csv(r'E:\GitProjects\StreamCat\ControlTable_StreamCatv2.csv')
```
Comment (debboutr) [inline ListPublishedUnpublishedFiles.py:25]: we should remove the v2 from the title of this file and add `index=False` so we don't get that extra column.

File: StreamCat.py [hunk 2]
```diff
@@ -102,30 +102,30 @@
...
-                )
-            if row.accum_type == "Point":
-                izd = f"{pre}/NHDPlusCatchment/Catchment.shp"
-                cat = PointInPoly(
-                    points, zone, izd, pct_full, mask_dir, apm, summaryfield
-                )
-            cat.to_csv(f"{OUT_DIR}/{row.FullTableName}_{zone}.csv", index=False)
-            in2accum = len(cat.columns)
-    print("Cat Results Complete in : " + str(dt.now() - catTime))
+    #     if not os.path.exists(f"{OUT_DIR}/{row.FullTableName}_{zone}.csv"):
```
Comment (debboutr) [inline StreamCat.py:105]: this isn't going to work. Why don't we want to do cat stats here? Did you spin up the cat stat files yourself for slope? as long as they are at eh path above this section would get skipped anyway

File: ListPublishedUnpublishedFiles.py (hunk 1 above)
Comment (mhweber) [reply ListPublishedUnpublishedFiles.py]: Thanks for reminder - meant to fix both of those two things, pushed change with that fix.

File: StreamCat.py (hunk 2 above)
Comment (mhweber) [reply StreamCat.py]: This was just me forgetting to uncomment section of code when I was doing a custom run - pushed up commit uncommenting that section.

**Pull request merged**